src/UITests/UILibrary/AIWinDriver.cs

```diff
@@ -50,7 +50,9 @@ public int ScanAIWin(TestContext context)
             var result = scanner.Scan();
             if (result.ErrorCount > 0)
             {
-                context.AddResultFile(result.OutputFile.A11yTest);
+                var newPath = Path.Combine(outputPath, $"{fileName}.a11ytest");
```

Comment:
FxCop may complain about this. The fix is:
```suggestion
                var newPath = Path.Combine(outputPath, Invariant($"{fileName}.a11ytest"));
```

and to add the following at the top of the file (line 9, maybe):

using static System.FormattableString;

Reply:
updated, thanks for the suggestion